CMakeLists.txt

```diff
@@ -277,6 +277,7 @@ if (X86_ENABLED)
         avx2-i8x32 avx2-i16x16
         avx2-i32x4 avx2-i32x8 avx2-i32x16 avx2-i64x4
         avx512knl-x16
+        # avx512spr targets are alised to avx512skx targets with attributes enabling fp16 and other new instructions.
```

Comment:
alised->aliased

Reply:
I've reimplemented the targets to make them non-alias, this line is gone now.